ludwig/models/llm.py

```diff
@@ -563,6 +563,8 @@ def save(self, save_path):
         # avoid this hack
         if self.config_obj.trainer.type != "none":
             weights_save_path = os.path.join(save_path, MODEL_WEIGHTS_FILE_NAME)
+            # We initialize the model's generation configuration; otherwise, we get a validation error.
+            self.model.generation_config = self.generation
```

Comment:
@arnavgarg1 Yes, and they also changed the `ViTModel` (I filed an issue: https://github.com/huggingface/transformers/issues/30978).  Unless you recommend changes, may I please get an approval so that @ethanreidel can continue his work?  Thanks!